static/elements/chromedash-form-field.js

```diff
@@ -65,6 +81,47 @@ export class ChromedashFormField extends LitElement {
           )}
         </sl-select>
       `;
+    } else if (type === 'input') {
+      let inputType = 'text';
+      let title = '';
+      let placeholder;
+      let pattern;
+      switch (fieldProps.input_type) {
```

Comment:
I would like every switch statement to have a default clause that logs some kind of an error for unexpected data.  The means you would need to make the `'text'` case explicit as a case in the switch too.   Having a default that flags an error would avoid a lot of time debugging a typo in the spec file.

Reply:
I suppose as an alternative, we could write a unit test  to validate the spec file.

Reply:
@jrobbins I made some changes, please review it again.

As you suggested, I added the `'text'` case as well as a default clause that will output an error if the type doesn't match of any listed. I think rather than writing a unit test now, I'd prefer to add it when a specific `<chromedash-input>` component is created.

In addition, I also made a slight change relevant to your recent PR #2143. I found that `value=field.initial` when value=None in guideforms.py does not work properly. I found it because the owners field in guide/new page should have a default value of user.email. I had to change `value=field.initial` to `value=bf.initial` to make it works.